ddtrace/runtime/__init__.py

```diff
@@ -31,7 +31,7 @@ def enable(tracer=None, dogstatsd_url=None, flush_interval=None):
         :param flush_interval: The flush interval.
         """
 
-        ddtrace.internal.runtime.runtime_metrics.RuntimeWorker.enable(
+        ddtrace.internal.runtime.runtime_metrics.RuntimeWorker.enable(  # type: ignore[attr-defined]
```

Comment:
why did this come up as part of this PR?

Reply:
🤷  is all I have at the moment I'm afraid 🙁 . I'll see if I can figure it out.

Reply:
I think we figured this out with the runtime metrics issue we recently fixed :(. These type hint comments are no longer required. Still wonder why they were not necessary before 🤔